user_deprovision/user_deprovision.py

```diff
@@ -431,8 +432,14 @@ def delete_user(access_token, user_email, from_email, prompt_del, auto_resolve,
 
 
 def main(args):
-    n_u = len(args.user_emails)
-    print("%d users to delete: %s"%(n_u, ', '.join(args.user_emails)))
+    email_list = []
+    for (i, item) in enumerate(csv.reader(args.user_csv)):
+        # Skip blank rows
+        if not item[0].strip():
+            continue 
+        email_list.append( item[0].strip() )
+    n_u = len(email_list)
+    print("%d users to delete: %s"%(n_u, ', '.join(email_list)))
```

Comment:
Let's also remove some legacy code while we're at it since `n_u` is not a great variable name. I would just call `len(email_list)` directly in the print function.